widgetssdk/src/main/res/drawable/ic_flip_visitor_camera.xml

```diff
@@ -0,0 +1,9 @@
+<vector xmlns:android="http://schemas.android.com/apk/res/android"
```

Comment:
Is this final from Risto or is this temporary for now?

Reply:
Risto said the icon is already done. So I think yes.